src/panels/ClusterPropertiesPanel.ts

```diff
@@ -260,5 +260,7 @@ function asKubernetesVersionInfo(version: KubernetesVersion): KubernetesVersionI
     return {
         version: version.version || "",
         patchVersions: version.patchVersions ? Object.keys(version.patchVersions) : [],
+        capabilities: version.capabilities ? Object.values(version.capabilities?.supportPlan || {}) : [],
```

Comment:
Would the "?" in version.capabilities?.supportPlan be redundant?

Reply:
🛡️ 💡  Thank you @ReinierCC for this comment, I see what you mean, yes indeed it does, because `capabilities` is **already** `nullable` checked,  I think inside if we really want to check it will be `supportPlan`. Likely a slipped the `?` in wrong place.

```*.ts
capabilities: version.capabilities ? Object.values(version.capabilities.supportPlan? || {}) : [],
```

* 🫶  Just minor thing this brings is the if we are returning the `supportPlan` then it will good to change the object name from `capabilities` to `supportPlan`.